### ISSUE TITLE
__VUE_PROD_DEVTOOLS__ missing in Vue 2 + webpack / Nuxt 2

### ISSUE THREAD

**Issue by posva** [posva/pinia#1570]
Currently, a project with Nuxt 2 won't have this Vue 3 flag defined.

from 

- https://github.com/vuejs/pinia/discussions/1567

**Comment by posva** [comment]
This can be fixed by adding the replace plugin to the webpack config:

```js
import webpack from 'webpack'
// or
const webpack = require('webpack')
// ...

  plugins: [
      new webpack.DefinePlugin({
        __VUE_PROD_DEVTOOLS__: 'false',
      }),
    ],
```

- With Nuxt, the `plugins` array goes into the `build` property in `nuxt.config.js`.
- With Vue CLI, the `plugins` array goes into the `configureWebpack` property in `vue.config.js`

**Comment by posva** [comment]
Fixed at 9468137
